### DISCUSSION TITLE
Only 1 of 3 OpenEBS ndm and csi pods being created on install

### DISCUSSION THREAD

**Discussion by brentgroves** [openebs/openebs#3609] [Q&A] [answered]
**Setup**
- The nodes are 3 ubuntu 22.04 server VM installed on VMware ESXi, 6.7.0 PowerEdge R710 server.
- microk8s channel 1.26/stable
- OpenEBS community add-on

**The issue**  
I think the reason openebs-jiva is not working is only 1 ndm,cstor-csi, and jiva-csi pod expected 3 of each.
(base)  brent@reports41  ~  kubectl get all -n openebs                                  
NAME                                                 READY   STATUS    RESTARTS   AGE
pod/openebs-ndm-lfr9m                                1/1     Running   0          103m
pod/openebs-jiva-operator-594fdd69b9-6gtck           1/1     Running   0          103m
pod/openebs-cstor-cvc-operator-7946b467f5-jdkpx      1/1     Running   0          103m
pod/openebs-cstor-admission-server-f5d57f788-qsmzs   1/1     Running   0          103m
pod/openebs-ndm-operator-74fc47c6cc-c5b74            1/1     Running   0          103m
pod/openebs-localpv-provisioner-99449bb55-zqn7r      1/1     Running   0          103m
pod/openebs-cstor-cspc-operator-7dd775b4b8-26kh7     1/1     Running   0          103m
pod/openebs-cstor-csi-node-t9n2l                     2/2     Running   0          103m
pod/openebs-jiva-csi-node-hl268                      3/3     Running   0          103m
pod/openebs-cstor-csi-controller-0                   6/6     Running   0          103m
pod/openebs-jiva-csi-controller-0                    5/5     Running   0          103m

**Comment by avishnu** [comment, marked as answer]
Recommend you to use Mayastor instead of Jiva/CStor, as they have reached EOL.
